dev/io.openliberty.xmlws.3.0.internal.tools/src/com/sun/tools/ws/wscompile/WsimportTool.java

```diff
@@ -0,0 +1,616 @@
+        //generated code
+        if (!options.quiet)
+            listener.message(WscompileMessages.WSIMPORT_GENERATING_CODE());
+
+        TJavaGeneratorExtension[] genExtn = ServiceFinder.find(TJavaGeneratorExtension.class).toArray();
+        CustomExceptionGenerator.generate(wsdlModel, options, receiver);
+        SeiGenerator.generate(wsdlModel, options, receiver, genExtn);
+        if (receiver.hadError()) {
+            throw new AbortException();
+        }
+        if (generateService) {
+            ServiceGenerator.generate(wsdlModel, options, receiver);
+        }
+        for (GeneratorBase g : ServiceFinder.find(GeneratorBase.class)) {
+            g.init(wsdlModel, options, receiver);
+            g.doGeneration();
+        }
+
+        List<String> implFiles = null;
+        if (options.isGenerateJWS) {
+            implFiles = JwsImplGenerator.generate(wsdlModel, options, receiver);
+        }
+
+        for (Plugin plugin : options.activePlugins) {
+            try {
+                plugin.run(wsdlModel, options, receiver);
+            } catch (SAXException sex) {
+                // fatal error. error should have been reported
+                return false;
+            }
+        }
+
+        if (options.getModuleName() != null) {
+            options.getCodeModel()._prepareModuleInfo(options.getModuleName(), JAXWS_MODULE);
+        }
+
+        CodeWriter cw;
+        if (options.filer != null) {
+            cw = new FilerCodeWriter(options);
+        } else {
+            cw = new WSCodeWriter(options.sourceDir, options);
+        }
+
+        if (options.verbose)
+            cw = new ProgressCodeWriter(cw, out);
+        options.getCodeModel().build(cw);
+
+        if (options.isGenerateJWS) {
+            //move Impl files to implDestDir
+            return JwsImplGenerator.moveToImplDestDir(implFiles, options, receiver);
+        }
+
+        return true;
+    }
+
+    public void setEntityResolver(EntityResolver resolver) {
+        this.options.entityResolver = resolver;
+    }
+
+    protected boolean compileGeneratedClasses(ErrorReceiver receiver, WsimportListener listener) {
+        List<String> sourceFiles = new ArrayList<String>();
+
+        for (File f : options.getGeneratedFiles()) {
+            if (f.exists() && f.getName().endsWith(".java")) {
+                sourceFiles.add(f.getAbsolutePath());
+            }
+        }
+
+        if (sourceFiles.size() > 0) {
+            String classDir = options.destDir.getAbsolutePath();
+            String classpathString = createClasspathString();
+            // Start Liberty Change
+            int majorJavaVersion = getMajorJavaVersion();
+            boolean bootCP = useBootClasspath(EndpointContext.class) || useBootClasspath(JAXBPermission.class) || useBootClasspath(WebService.class);
+            // End Liberty Change
+            List<String> args = new ArrayList<String>();
+
+            args.add("-d");
```

Comment:
I question any of this being needed because we don't need to update the boot classpath since everything will be on the classpath and nothing in the JDK will override it which is why we were prepending to the classpath.  As such I wonder if this works without us doing any updates in an overlay.